src/Sulu/Bundle/ContentBundle/Resources/public/js/components/smart-content/categories/main.js

```diff
@@ -138,7 +138,7 @@ define(['services/husky/util', 'services/sulucontent/user-settings-manager'], fu
                             instanceName: this.options.instanceName,
                             url: [
                                 '/admin/api/categories',
-                                '?locale=' + UserSettingsManager.getContentLocale(this.options.webspace),
+                                '?locale=' + this.sandbox.sulu.user.locale,
```

Comment:
That's not what we want. The categories should always be displayed in the language of the content, because that's also what the end result on the website will look like.